test/torch/model/test_simple_torch_model.py

```diff
@@ -206,3 +207,23 @@ def test_simple_model():
     evaluator = Evaluator(quantiles=[0.5, 0.9], num_workers=None)
 
     agg_metrics, _ = evaluator(ts_it, forecast_it)
+
+    # equality check
+    another_training_splitter = InstanceSplitter(
+        target_field=FieldName.TARGET,
+        is_pad_field=FieldName.IS_PAD,
+        start_field=FieldName.START,
+        forecast_start_field=FieldName.FORECAST_START,
+        instance_sampler=ExpectedNumInstanceSampler(
+            num_instances=2,
+            min_future=prediction_length,
+        ),
+        past_length=context_length,
+        future_length=prediction_length,
+        time_series_fields=[FieldName.OBSERVED_VALUES],
+    )
+
+    another_predictor = net.get_predictor(
+        transformation + another_training_splitter
+    )
+    assert not equals(predictor, another_predictor)
```

Comment:
I think it would make sense to also check for equality.

Reply:
added